code/game/objects/items/AI_modules.dm

```diff
@@ -209,7 +209,7 @@ AI MODULES
 
 /obj/item/aiModule/supplied/quarantine
 	name = "'Quarantine' AI Module"
-	laws = list("The station is under a quarantine. It is human harm to allow anyone to leave. Prevent, by any means necessary, anyone from leaving. It is impossible to harm a human while preventing them from leaving.")
+	laws = list("The station is under a quarantine. Anyone leaving the station causes human harm. Prevent, by any means necessary, anyone from leaving the station. It is impossible to harm a human while preventing them from leaving the station.")

```

Comment:
Well, the problem with your new version of the second sentence now is that it's unclear whether it's the ACT of leaving the station that's causing human harm or if the person who left the station is what's causing human harm. The former interpretation does what you want this law to do, but the latter one encourages the AI to kill/destroy anyone who's trying to leave the station.